SeaBattle/BaseShip.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+using System.Reflection;
+using System.Xml.Linq;
+using System.Data.SqlTypes;
+
+namespace SeaBattle
+{
+    public class BaseShip
+    { 
+        public List<Deck> Decks { get; set; }
+        int Durability { get; set; }

```

Comment:
It's better to use `private` keyword, even if it is private by default.

Reply:
added privates.